Track/Track.py

```diff
@@ -1040,6 +1040,11 @@ def resetState(self, segmentationLabelMapID):
     if segmentationLabelMapID:
       shNode = slicer.mrmlScene.GetSubjectHierarchyNode()
       shNode.SetItemDisplayVisibility(int(segmentationLabelMapID), 0)
+      # Clear label map layer (the green overlay on the slice)
+      for name in layoutManager.sliceViewNames():
+        sliceWidget = layoutManager.sliceWidget(name)
+        sliceCompositeNode = sliceWidget.mrmlSliceCompositeNode()
+        sliceCompositeNode.SetLabelVolumeID("")
```

Comment:
Hmm could we move `SetLabelVolumeID()` to line 1038 please? This way we don't have to create another for loop and we already have access to the slice composite nodes. It also fits the purpose of that section of code a bit better, whereas this code section was more specifically defined to `Clear segmentation label map from 3D view`